pyband/transaction.py

```diff
@@ -57,11 +44,10 @@ def with_gas(self, gas: int) -> "Transaction":
     def with_memo(self, memo: str) -> "Transaction":
         if len(memo) > MAX_MEMO_CHARACTERS:
             raise ValueTooLargeError("memo is too large")
-
         self.memo = memo
         return self
 
-    def get_sign_data(self) -> bytes:
+    def get_tx_data(self, privateKey: PrivateKey) -> bytes:
```

Comment:
Is it better if the function receive signature and pubkey instead of private key? In case that user would like to sign the transaction elsewhere, such as hardware-wallet.